.vib/cluster-autoscaler/goss/vars.yaml

```diff
@@ -0,0 +1,7 @@
+binaries:
+  - cluster-autoscaler
+directories:
+  - mode: "0775"
```

Comment:
We are not setting permissions for this folder, could you please remove the `mode` check?